registration/models.py

```diff
@@ -243,6 +243,7 @@ def first_activated(self):
         """
         if self.instance:
             return self.instance.first_activated
+        return None
```

Comment:
Is it `pylint` which want to use `return None` instead of just `return` to be more explicit?

Reply:
Yes